### DISCUSSION TITLE
Struggling to open a URL in Chrome (or any other browser)

### DISCUSSION THREAD

**Discussion by EmmCeeSquared** [ttscoff/bunch-docs#333] [Q&A] [answered]
![CleanShot 2024-07-19 at 14 53 15](https://github.com/user-attachments/assets/e80317ae-ad39-4b0c-9e41-5358b063245f)

Sorry for the basic question - struggling to open a URL despite trying every combination of Chrome's name. What am I doing wrong? It'll be user error. I've made sure Bunch has full disk access and I've tried calling chrome everything - Google Chrome, Chrome, Chromepleasework but no dice. 

Thanks for any tips.

**Comment by ttscoff** [comment, marked as answer]
The browser name needs to be lower case, e.g. 
`chrome:https://brettterpstra.com`.

-Brett

**Comment by EmmCeeSquared** [comment]
Brett, thanks so much. Autocorrect has done me wrong! I didn't even realise it was autocorrecting. It's working perfectly now, thanks so much. While I have your attention (I love Bunch and thanks for creating it), if it's not too cheeky could I ask for your top tips on getting bunch to foreground apps when they're closed/in the dock? At the moment, my workaround is to use Alfred to close a bunch I want to run first, before re-opening it. It's not a show-stopper, but it does feel a little clunky. Thanks again!

**Comment by ttscoff** [comment]
Do you mean you want to foreground apps when they're opened by a Bunch, 
or something else? You can force an app to foreground by following it 
with a carat (^), e.g. `Safari^`, but I think that will only work on 
open, and it's not perfect in its implementation (due to OS 
shortcomings).

-Brett

**Reply by EmmCeeSquared** [reply]
Thanks again Brett, yeah, I've been trying the carat, but as you say it's a little clunky. So long as I know it's not me, and it's a known issue, it's all good. Thanks again for your help, and brilliant app. Much appreciated.

**Comment by ke6yjc** [comment]
I'm trying to do the same thing, but I want to have each of the URL's launched in a new window so I can re-arrange each of the sites on my desktop. Is there a way to accomplish this? Thanks in Advance!

**Comment by ttscoff** [comment]
You would have to set your browser to open links in new windows. Bunch just sends an open command to the browser, how tabs/windows are handled depends on your settings.

- Brett
